NetKAN/SDHI-ServiceModuleSystem.netkan

```diff
@@ -13,19 +13,22 @@
     "install": [
       {
         "file": "GameData/SDHI",
-        "install_to": "GameData"
+        "install_to": "GameData",
+        "filter_regexp" : [
+               "Agencies"
```

Comment:
This will filter any file/directory with `Agencies` anywhere in it. A simple:
```json
{
    "filter": "Agencies"
}
```
ought to do.